MountSionCBSSecondary/SchoolWebsite
# Added GitHub Authorisation Tutorial to docs folder

I am adding this tutorial as I think it will benefit the newcomers to this project in the future.

# Description

New tutorial on how to authorize your computer for work on github has been added to the /docs folder. It has been added so that people will know how to authorize their computers for work on this project.

## What

A tutorial that wasn't previously available has been added for learning and teaching use.

## Why

This tutorial has been added because it will benefit future project workers, in my opinion. They will have easy access to a step-by-step process that will inevitably remove some stress from the setting up and continuation of the project.

## How

The ODF file was uploaded via GitHub and was added to a branch.

## Verification Steps

-Download file
-Open it using OpenOffice, LibreOffice, or Microsoft Office.
-Ensure all slides are present
-Leave a review or request below


# Checklist:
